tools/cli/main.js

```diff
@@ -616,7 +616,7 @@ Fiber(function () {
   // tight timetable for 1.0 and there is no advantage to doing it now
   // rather than later. #ImprovingCrossVersionOptionParsing
 
-  var isBoolean = { "--help": true, "--unsafe-perm": true };
```

Comment:
I think we should keep command-line flag. I think people know much better how to set that than how to run `METEOR_UNSAFE_PERM=true meteor`. But maybe `unsafe-perm` should just set `METEOR_UNSAFE_PERM` and this is it. :-)